.azure-pipelines.yml

```diff
@@ -0,0 +1,29 @@
+﻿# Use this to schedule a build at defined days and times
```

Comment:
Would it be cleaner to put this in the build folder? So that everything pipeline related is in the same place?

Reply:
I think it's best to have this file at the root of the project. Let me check if I can find anything.

Reply:
I will also remove the dot. See https://learn.microsoft.com/en-us/azure/devops/pipelines/customize-pipeline?view=azure-devops#understand-the-azure-pipelinesyml-file for more details.

Reply:
Done.

Reply:
I'd rather have it in the build folder. That way all the files associated to pipelines are in 1 place instead of 2.